mbuild/tests/test_compound.py

```diff
@@ -956,7 +956,12 @@ def test_from_pybel_monolayer(self):
         first_atom = monolayer.OBMol.GetAtom(1)
         assert np.allclose(cmpd[0].pos, [first_atom.GetX()/10, first_atom.GetY()/10, first_atom.GetZ()/10])
         #assert np.allclose(box.lengths,
-        #        [monolayer.unitcell.GetA()/10, monolayer.unitcell.GetB()/10, 
-        #            monolayer.unitcell.GetC()/10], 
+        #        [monolayer.unitcell.GetA()/10, monolayer.unitcell.GetB()/10,
+        #            monolayer.unitcell.GetC()/10],
         #        rtol=1e-3)
 
+    @pytest.mark.skipif(not has_openbabel, reason="Pybel is not installed")
```

Comment:
Can we test a few molecules? Nothing crazy but it would be nice to include a few more chemistries, i.e. a simple ring system and another branched molecule. Just a few SMILES strings for pytest to iterate over

Reply:
Sure, there are some compounds that break it when it messes up the double bonds but I can try and find a few more that work

Reply:
I'm running this test with a few molecules I added.  Like you said, the test fails with anything strings defining double bonds.  But I'm also getting a failure with this: `C1COCCO1.C1COCCO1`.

This doesn't mean that the functionality doesn't work, it just means that the SMILES string generated from `get_smiles` is sometimes different than the original SMILES string, even though they generate the same compound.

I think this is okay for now, but I think we want to further think about how we can compare two mbuild compounds (#64)

Reply:
For `C1COCCO1.C1COCCO1`, it looks like a smiles string for TWO molecules? pybel seems to correctly understand the smiles string is two molecules, but mbuild gets confused and thinks they're actually just one molecule
![image](https://user-images.githubusercontent.com/6175121/66083901-f4d16580-e532-11e9-96f5-59781d25a18b.png)

Reply:
Hm well that's embarrassing, I blindly grabbed these smiles strings from Matt's RDkit project.  But I think a smiles string with a double bond will cause this issue?

Reply:
The `.` in smiles means "disconnected structure" so I'm not surprised its weird, we don't handle that case on generating the compound.

Reply:
Ignoring the `.` I find that while the smiles in might not match the smiles out, the connectivity is right, and for the cases I looked at the hydrogen/bond order is correct, mostly the case of carbon is "off"

Reply:
Agreed, LGTM

Reply:
Hmm, I grabbed the SMILES strings from https://pubchem.ncbi.nlm.nih.gov/compound/1_4-dioxane#section=InChI. Not relevant for this PR but a good example of why doing work completely alone can lead to some pretty easy errors.